api/resources_portal/views/material_request.py

```diff
@@ -87,15 +88,50 @@ def send_material_request_notif(notif_type, request):
 
 def send_transfer_update_notif(status, request):
     if status == "APPROVED":
-        send_material_request_notif("TRANSFER_APPROVED", request)
+        send_material_request_notif("TRANSFER_APPROVED", request, request.requester)
     elif status == "REJECTED":
-        send_material_request_notif("TRANSFER_REJECTED", request)
+        send_material_request_notif("TRANSFER_REJECTED", request, request.requester)
     elif status == "FULFILLED":
-        send_material_request_notif("TRANSFER_FULFILLED", request)
+        send_material_request_notif("TRANSFER_FULFILLED", request, request.requester)
     else:
         return
 
 
+def user_owns_attachment(attachment, user):
+    if attachment.owned_by_org:
+        return user in attachment.owned_by_org.members.all()
+    else:
+        return False
+
+
+def user_in_attachment_org(attachment, user):
+    if attachment.owned_by_user:
+        return user == attachment.owned_by_user
```

Comment:
This check doesn't seem to match the function name. I think we can probably remove this function and just add this check to `user_owns_attachment` though.